doc/aaa/ldap.md

```diff
@@ -844,6 +844,12 @@ $ show ldap
 
 Similar to KLISH, but config, and show utilities.
 
+Note:
+
+1. AAA has a holdoff timer of 3 seconds to decrease the frequency of time intensive operation 
+   of updating AAA configuration files. Hence, AAA configurations will be updated in the respective
```

Comment:
Do we have the code already for this delay?

Reply:
yes code is present in broadcom_sonic

Reply:
@venkatmahalingam : Can you please approve this ?